src/utils/macros.rs

```diff
@@ -201,3 +201,22 @@ macro_rules! impl_from {
         }
     };
 }
+
+#[cfg(test)]
+mod test {
+    #[test]
+    fn test_queue() {
+        use crate::utils::command::Output;
+        use std::io::{stdout, Write};
+        assert!(queue!(stdout(), Output("hi"),).is_ok());
```

Comment:
Please, do not use `Output`. We do not want to print anything on `stdout`. Instead of this, create a fake command in the `test` module. Something like:

```rust
pub struct FakeCommand;

impl Command for FakeCommand {
    type AnsiType = &'static str;

    fn ansi_code(&self) -> Self::AnsiType {
        ""
    }

    #[cfg(windows)]
    fn execute_winapi(&self) -> Result<(), ErrorKind> {
        Ok(())
    }
}
```